src/component/common/ProtectedRoute/ProtectedRoute.jsx

```diff
@@ -1,17 +1,21 @@
-import { Route, Redirect } from 'react-router-dom';
+import { Route, useLocation, useHistory } from 'react-router-dom';
 
 const ProtectedRoute = ({
     component: Component,
     unauthorized,
     renderProps = {},
     ...rest
 }) => {
+    const { pathname } = useLocation();
+    const history = useHistory();
+    const loginLink =
+        pathname.length > 1 ? `/login?redirect=${pathname}` : '/login';
```

Comment:
Could we always add the param? Should test this both with and without a baseUriPath.

Reply:
Should we remove the length condition, and always add the query param? Seems simpler.